src/app/public/modules/phone-field/phone-field.component.html

```diff
@@ -0,0 +1,45 @@
+<div
+  class="sky-form-group sky-input-group"
+>
+  <div
+    class="sky-input-group-btn sky-phone-field-country-btn"
+  >
+    <sky-dropdown
+      [disabled]="disabled"
+      [label]="'skyux_phone_field_dropdown_label' | skyLibResources"
+      [title]="selectedCountry.name + ' +' + selectedCountry.dialCode"
+    >
+      <sky-dropdown-button>
+        <div
+          [className]="'iti-flag ' + selectedCountry.iso2"
+        >
+        </div>
+      </sky-dropdown-button>
+      <sky-dropdown-menu>
+        <sky-dropdown-item *ngFor="let country of countryData; let i = index"
+          [ngClass]="{
+            'sky-phone-field-country-selected': i === 0,
+            'sky-phone-field-country-default': country.iso2 === defaultCountry
+          }"
+        >
+          <button
+            type="button"
+            (click)="selectCountry(country.iso2)"
```

Comment:
Callback function should be prefixed with `on` to match Angular's style guide: https://angular.io/guide/styleguide#dont-prefix-output-properties

Reply:
Done. I changed `selectCountry` to be `onCountrySelected`